packages/node_modules/@ciscospark/common/test/unit/spec/one-flight.js

```diff
@@ -5,6 +5,43 @@ import {assert} from '@ciscospark/test-helper-chai';
 
 describe(`common`, () => {
   describe(`@oneFlight`, () => {
+    it(`returns new promise on function with undefined keyFactory`, () => {
```

Comment:
What happens if you call a oneflighted function twice without a keyfactory? isn't that the scenario that was causing you a problem?